packages/salesforcedx-vscode-core/test/vscode-integration/conflict/metadataCacheService.test.ts

```diff
@@ -322,16 +324,18 @@ describe('Metadata Cache', () => {
         name: 'CustomField'
       }
     };
-    const fileProperties = [{
-      fullName: 'HandlerCostCenter',
-      lastModifiedDate: 'Today',
-      type: 'ApexClass'
-    },
-    {
-      fullName: 'Account',
-      lastModifiedDate: 'Yesterday',
-      type: 'CustomObject'
-    }] as FileProperties[];
+    const fileProperties = [
```

Comment:
nit: does `const: Array<FileProperties> = [{` work?

Reply:
Looks like some props are missing:
<img width="500" alt="2022-08-25_14-49-04" src="https://user-images.githubusercontent.com/46458081/186775316-59cb1866-49b9-475d-ae08-55b18d5e2462.png">
<img width="500" alt="2022-08-25_14-49-40" src="https://user-images.githubusercontent.com/46458081/186775327-3ef17034-068a-4a63-bb65-655ad9abfa8d.png">

Reply:
Weird: I swear the linter just yelled at me to use Array<T> vs T[] but whatever.  
Hum....if you wanted to leave this cleaner than you found it you could add the properties and remove the `as`.  By saying `as FileProperties[]` we are telling typescript we know better then it does what type the object is which can lead to badness where we _think_ that a property has to exist on an object of the type but then it reality it doesn't.  I get this is a test file and it may be less painful to allow `as` at will in tests to keep the LOC down.  I'm open to discussion about it.

Reply:
@klewis-sfdc thoughts on this one?

Reply:
Updated! let me know how it looks